src/NuGet.Core/NuGet.Configuration/Settings/Settings.cs

```diff
@@ -378,7 +420,14 @@ private static ISettings LoadSettingsGivenSettingsFiles(List<SettingsFile> setti
             {
                 validSettingFiles.AddRange(
                     GetSettingsFilesFullPath(root)
-                        .Select(f => ReadSettings(root, f))
+                        .Select(f =>
```

Comment:
If you follow the stack trace, I don't think you are handling the user wide settings. 

https://github.com/NuGet/NuGet.Client/blob/ba7cbdae5fe833d1b94f833ac9de0c9f33acd6d5/src/NuGet.Core/NuGet.Configuration/Settings/Settings.cs#L436

Same thing applies if the machine wide settigns are not provided. 

https://github.com/NuGet/NuGet.Client/blob/ba7cbdae5fe833d1b94f833ac9de0c9f33acd6d5/src/NuGet.Core/NuGet.Configuration/Settings/Settings.cs#L446

Reply:
Does my next commit look better?  I _think_ I got them all now...

Reply:
I think it looks good!